src/style.css

```diff
@@ -0,0 +1,185 @@
+/* #region reset */
+
+* {
+  box-sizing: border-box;
+}
+
+html,
+body,
+div,
+p,
+a {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+/* #region card */
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  margin: auto;
+  margin-top: 16px;
+  margin-bottom: 40px;
+}
+
+.card__product-name {
+  padding-bottom: 4px;
+
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px;
+  font-size: 12px;
+  color: #060b35;
+}
+
+.card__product-code {
+  padding-bottom: 16px;
+
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  font-size: 10px;
+  color: #616070;
+}
+
+/* #region stars */
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+
+  padding-bottom: 24px;
+}
+
+.stars__rate-stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+
+  list-style: none;
+}
+
+.stars__rate-stars :last-child {
+  margin-right: 0;
+}
+
+.stars__rate-stars-star {
+  position: relative;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(images\star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__rate-stars-star:nth-child(-n+4) {
+  background-image: url(images\star-active.svg);
+}
```

Comment:
You should add all possible modifiers for the stars block, that you've created in previous task
And in html just need to add corresponding modifier to specify which number of stars should be active